### PR TITLE
feat: add metadata sink for file

### LINKED ISSUE #129: Add metadata sink for a file
Sink final results to a file

### PR DESCRIPTION
**Pull request by GrayFlash**
closes #129

### CODE REVIEW

**Comment by GrayFlash** [comment]
@StewartJingga @mabdh can you folks check the `sample.json` and `sample.yaml` files once, and validate if the output format seems correct, or will need any changes??

File: plugins/sinks/file/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    name: file
+    config:
+        path: "./dir/sample.yaml"
+```
+
+## Config Defination
+
+| Key | Value | Example | Description |  |
+| :-- | :---- | :------ | :---------- | :-- |
+|`path` | `string` | `./dir/sample.yaml` | path to output file, directory should exist, file should be either yaml or json  | *required*|
```
Comment (mabdh) [inline plugins/sinks/file/README.md:18]: Is the path relative to binary / home folder?

File: plugins/sinks/file/file.go [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+	format string
+}
+
+func New() plugins.Syncer {
+	return new(Sink)
+}
+
+func (s *Sink) Info() plugins.Info {
+	return plugins.Info{
+		Description:  "save output to file",
```
Comment (mabdh) [inline plugins/sinks/file/file.go:42]:
```suggestion
		Description:  "save output to a file",
```

File: plugins/sinks/file/README.md (hunk 1 above)
Comment (GrayFlash) [reply plugins/sinks/file/README.md]: It is relative to binary in test, but even an absolute file path w.r.t home should work

Comment (mabdh) [reply plugins/sinks/file/README.md]: maybe better to add some details about that in the readme @GrayFlash

Comment (GrayFlash) [reply plugins/sinks/file/README.md]: ok doing it

**Pull request merged**